leapp/cli/report/__init__.py

```diff
@@ -0,0 +1,40 @@
+"""
+This module is currently single workflow specific (upgrades). After introducing other use cases we can make it
+more robust
+"""
+
+from __future__ import print_function
+import os
+import sys
+
+from leapp.exceptions import CommandError
+from leapp.utils.clicmd import command, command_opt, command_arg
+from leapp.utils.report import fetch_upgrade_report_messages
+from leapp.cli.upgrade import fetch_last_upgrade_context
+
+
+@command('report', help='Create report for last upgrade execution (default) or specific execution id')
+@command_opt('--id', help='ID of particular run to print report for. See # leapp list-runs for a list of all runs')
+@command_opt('--format', help='Format report using particular renderers. By default: "plaintext"')
+def report(args):
+    if os.getuid():
+        raise CommandError('This command has to be run under the root user.')
+
+    id = args.id or fetch_last_upgrade_context()[0]
+
+    if not id:
+        raise CommandError('No previous Leapp upgrade run of the requested context {} found'.format(id))
```

Comment:
That would give you the message:

> No previous Leapp upgrade run of the requested context None found

Maybe better something along these lines:
> No previous Leapp upgrade run found. This command can only be run after 'leapp upgrade' has been executed.

Reply:
you are right...